control/mateqn.py

```diff
@@ -41,16 +41,17 @@
 Author: Bjorn Olofsson
 """
 
-from scipy import shape, size, asarray, asmatrix, copy, zeros, eye, dot
+from scipy import shape, size, array, asarray, copy, zeros, eye, dot
```

Comment:
again not the point, but since you remove `import scipy as sp`, I think this should be `from numpy import ...`